doc/ref_model/chapters/chapter01.md

```diff
@@ -83,7 +83,7 @@ This section introduces the high-level principles of infrastructure abstraction
 
 1. A top-level objective of the Common Telco NFVI is to build a single, overarching Reference Model with the smallest number of Reference Architectures tied to it as is practical. Two principles are introduced in support of these objectives:
     - **Minimize Architecture proliferation by stipulating compatible features be contained within a single Architecture as much as possible:**
-      - Features which are compatible, meaning they are not mutually exclusive and can coexist in the same NFVI instance, shall be incorporated into the same Reference Architecture. For example, IPv4 and IPv6 should be captured in the same Architecture, because they don't interfere with each other
+      - Features which are compatible, meaning they are not mutually exclusive and can coexist in the same NFVI Profile, shall be incorporated into the same Reference Architecture. For example, IPv4 and IPv6 should be captured in the same Architecture, because they don't interfere with each other
```

Comment:
I think NFVI profile is not the correct term here as multiple NFVI Profiles are supported within the same NFVI.  NFVI Profile is not yet defined. The confusion may have come from the example used.

In this section "features" needs to be about core components of NFVI for example the "virtualisation layer" or "abstractions" that NFVI provides to the workloads.

Reply:
I think NFVI instance was the original intent (i.e. the specific NFVI deployment can support both IPv4 and IPv6, but it may not make sense to combine OpenStack and Kubernetes into the same conceptual NFVI as the manner of configurations, management, concepts are conflicting.

Reply:
I would rather use "NFVI deployment", as "NFVI Instance" is overloaded in the document.